src/components/hero/campaignHero.tsx

```diff
@@ -1,24 +1,44 @@
 import { Hero, Link as DSLink } from "@nypl/design-system-react-components";
-import { imageURL } from "../../utils/utils";
 import CampaignHeroSubText from "./campaignHeroSubText";
 import CampaignHeroHeading from "./campaignHeroHeading";
+import CampaignHeroLoading from "./campaignHeroLoading";
+import { useEffect, useState } from "react";
 
-const CampaignHero = ({ featuredItem, numberOfDigitizedItems }: any) => {
-  return (
+const CampaignHero = ({ imageID }) => {
+  console.log(imageID);
+  const [data, setData] = useState<any>({});
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await fetch(`/api/featuredHero?imageID=${imageID}`);
+      const responseData = await response.json();
+      console.log(responseData);
+      setData(responseData);
+    };
+
+    console.log("fetching hero data");
+    fetchData();
+  });
```

Comment:
This is causing an infinite loop but not really sure why. What works is to add a second parameter, the dependency array with `imageID`. So on line 21, `, [imageID]);`

Reply:
I noticed this on the terminal but you can see this in the browser's console as well.